plugins/woocommerce/templates/single-product/add-to-cart/variable.php

```diff
@@ -45,13 +45,27 @@
 										'product'   => $product,
 									)
 								);
-								echo end( $attribute_keys ) === $attribute_name ? wp_kses_post( apply_filters( 'woocommerce_reset_variations_link', '<a class="reset_variations" href="#">' . esc_html__( 'Clear', 'woocommerce' ) . '</a>' ) ) : '';
+							?>
+						</td>
+					</tr>
+					<tr>
+						<td colspan="2">
+							<?php
+							/**
+							 * Filters the reset variation button.
+							 *
+							 * @since 2.5.0
+							 *
+							 * @param string  $button The reset variation button HTML.
+							 */
+							echo end( $attribute_keys ) === $attribute_name ? wp_kses_post( apply_filters( 'woocommerce_reset_variations_link', '<button class="reset_variations"  aria-label="' . esc_html__( 'Clear options', 'woocommerce' ) . '">' . esc_html__( 'Clear', 'woocommerce' ) . '</button>' ) ) : '';
 							?>
 						</td>
 					</tr>
 				<?php endforeach; ?>
 			</tbody>
 		</table>
+		<div class="reset_variations_alert screen-reader-text" role='alert' aria-live='polite' aria-relevant="all"></div>
```

Comment:
For consistency sake, we should use double quotes on all attributes:

```suggestion
		<div class="reset_variations_alert screen-reader-text" role="alert" aria-live="polite" aria-relevant="all"></div>
```